### PR TITLE
Fixed issue in SS T-Test posterior plots due to BF type flip

### PR DESCRIPTION
**Pull request by akashrajkn**
Don't recompute Prior Posterior plot if BF type changes

### CODE REVIEW

File: JASP-Engine/JASP/R/commonsummarystatsttestbayesian.R [hunk 1]
```diff
@@ -202,7 +202,7 @@
   if (is.null(container)) {
     container <- createJaspContainer()
     # add dependencies for main table (i.e., when does it have to recompute values for the main table)
-    container$dependOn(c("tStatistic", "n1Size", "n2Size", "hypothesis", "bayesFactorType",            # standard entries
```
Comment (TimKDJ) [inline JASP-Engine/JASP/R/commonsummarystatsttestbayesian.R:205]: The robustness check does need a dependency on the `bayesFactorType`.
The flipping of axes I spoke of earlier was about this plot apparently ;)

**Review by TimKDJ — CHANGES_REQUESTED** [review]

File: JASP-Engine/JASP/R/commonsummarystatsttestbayesian.R (hunk 1 above)
Comment (boutinb) [reply JASP-Engine/JASP/R/commonsummarystatsttestbayesian.R:205]: @TimKDJ Can you fix this issue?

Comment (boutinb) [reply JASP-Engine/JASP/R/commonsummarystatsttestbayesian.R:205]: @akashrajkn I am not so sure anymore what we have to do with this PR. Tim is ill. Should we merge it?

Comment (akashrajkn) [reply JASP-Engine/JASP/R/commonsummarystatsttestbayesian.R:205]: @boutinb I need to make a change in this. Can you wait till tomorrow?

Comment (boutinb) [reply JASP-Engine/JASP/R/commonsummarystatsttestbayesian.R:205]: OK

Comment (boutinb) [reply JASP-Engine/JASP/R/commonsummarystatsttestbayesian.R:205]: @akashrajkn Can you have a look today? Then we can close this issue.

Comment (AlexanderLyNL) [reply JASP-Engine/JASP/R/commonsummarystatsttestbayesian.R:205]: @akashrajkn have you looked into this now?

**Comment by boutinb** [comment]
Hi @akashrajkn Should we close this PR?

**Comment by boutinb** [comment]
@akashrajkn will make another PR later

**Pull request closed without merging**